addons/sourcemod/scripting/PlayerVisibility.sp

```diff
@@ -169,61 +169,59 @@ public void OnClientDisconnect(int client)
 public MRESReturn AcceptInput(int pThis, Handle hReturn, Handle hParams)
 {
 	// Should not happen?
-	if(DHookIsNullParam(hParams, 2))
+	if (DHookIsNullParam(hParams, 2))
 		return MRES_Ignored;
 
 	int client = EntRefToEntIndex(DHookGetParam(hParams, 2));
-	if(client < 1 || client > MAXPLAYERS)
+	if (client < 1 || client > MAXPLAYERS)
 		return MRES_Ignored;
 
-	if(!g_playerData[client].enabled)
+	if (!g_playerData[client].enabled)
 		return MRES_Ignored;
 
 	char szInputName[32];
 	DHookGetParamString(hParams, 1, szInputName, sizeof(szInputName));
 
-	if(!StrEqual(szInputName, "addoutput", false))
-		return MRES_Ignored;
-
-	char sValue[128];
-	DHookGetParamObjectPtrString(hParams, 4, 0, ObjectValueType_String, sValue, sizeof(sValue));
-	int iValueLen = strlen(sValue);
+	// Handle different input types
+	if (StrEqual(szInputName, "addoutput", false))
```

Comment:
Would it be better to use `strcmp` instead of `strequal` for marginal performance improvement?